how-to/how-to-use-rootless-kata-containers-with-podman.md

```diff
@@ -0,0 +1,275 @@
+    $ sudo bash -c "echo $(whoami):110000:65536 > /etc/subuid"
+    $ sudo bash -c "echo $(whoami):110000:65536 > /etc/subgid"
+    ```
+
+- Golang 1.12
+
+## Installation
+
+Refer to the following table for the __minimum__ required software versions
+and the installation instructions:
+
+| Component       | Version       | Install Instructions|
+| ----------------|:-------------:|---------------------|
+| Podman          | 1.6.2         | [see here](https://github.com/containers/libpod/blob/master/install.md)
+| `slirp4netns`   | 0.4.0         | [see here](https://github.com/rootless-containers/slirp4netns#quick-start)
+| Kata Containers | 1.10.0-alpha1 | [see here](https://github.com/kata-containers/documentation/blob/master/install/README.md)
+
+
+> **NOTE:**
+>
+> If installing Podman with a package manager, there is usually no need to
+> install `slirp4netns` separately.
+> You will need to install Kata Containers from source today for rootless support.
+> Rootless support for Kata should be available and functional in upcoming 1.9.2 and 1.10.0-alpha1 releases.
+
+## Configuration
+
+Now that you have installed Kata Containers and Podman, you need to configure
+them for rootless execution.
+
+### Disable SELinux
+
+> **Warning:**
+>
+> You should understand the security implications before disabling SELinux.
+
+If SELinux is installed and enabled, it needs to be disabled with the
+following command (Kata Containers
+[does not support SELinux](https://github.com/kata-containers/documentation/blob/master/Limitations.md#selinux-support)).
+
+> **Warning:**
+> The following command might differ depending on the distro you use:
+
+```bash
+$ [ -f /etc/selinux/config ] && sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config
+```
+
+### Add user to KVM group
+
+If running a KVM based hypervisor, add the user running the workload to the KVM group:
+
+```bash
+$ sudo usermod -a -G kvm $USER
+```
+> **NOTE:**
+>
+> `kvm` should be the group owning the device node `/dev/kvm` on most distros.
+> Make sure permissions on `/dev/kvm` are as shown below:
+> ```
+> $ ls -la /dev/kvm
+> crw-rw---- 1 root kvm 10, 232 Nov 11 20:28 /dev/kvm
+> ```
+> If the group owner happens to be `root`, you may need to create a system group `kvm` and
+> change permissions for `/dev/kvm` as shown above.
+> ```
+> $ sudo addgroup --system kvm
+> $ sudo chown root:kvm /dev/kvm
+> $ sudo chmod g+rw /dev/kvm
+> ```
+ 
+### Reboot
+
+Reboot the system for the changes to take effect (when you disable SELinux you
+must reboot, while logging out and back in is enough to have that user joining
+the `kvm` group).
+
+Verify the configuration is correct:
+
+- If installed, disable SELinux:
+  ```bash
+  $ getenforce
+  Disabled
+  ```
+
+- The user should be in the `kvm` group:
+
+  ```
+  $ groups | grep -ow kvm
+  kvm
+  ```
+
+### Setup Kata configuration files
+
+If the Kata `configuration.toml` file does not exist in `/etc`, create it:
```

Comment:
These instructions all fall apart if you used kata-deploy to install.   You have to edit  `/opt/kata/share/defaults/kata-containers/configuration-qemu.toml`    You could probably just say "These next set of instructions are based on Kata Containers being installed from a package manager. If `kata-deploy` was used, then the binaries and configuration.toml files will be located at `/opt/kata/` and the following instructions will have to be modified.